src/components/invest/investslug/components/AuthorSection.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+import { cn } from '@/lib/utils'
+import { AvatarImage } from '@/components/shared/AvatarImage'
+
+interface AuthorSectionProps {
+  author: string
+  imageURL: string
+  tags: string[]
+}
+
+export const AuthorSection: React.FC<AuthorSectionProps> = ({ author, imageURL, tags }) => {
+  return (
+    <section className='w-full py-2 md:py-8 flex flex-row gap-4 md:gap-10 justify-between items-center'>
+      <div className='flex items-center gap-2'>
+        <AvatarImage src={imageURL} size={42} alt='Author Avatar' className='' />
+        <div className="text-white text-base font-bold font-['Satoshi'] leading-snug">{author}</div>
```

Comment:
We should use either `span` or `p` (in the future we will use Text component) when we want to display some text, we should not use `div` for that purpose.

Also one note: Satoshi font is by default, only if you are using h1-h6 tags then it will use Obviously narrow font family.

Reply:
removed font-['Satoshi'], swaped <div> for <span>